tests/Collector/RouterCollectorTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Yiisoft\Yii\Debug\Tests\Collector;
+
+use Yiisoft\Di\Container;
+use Yiisoft\Router\Group;
+use Yiisoft\Router\Route;
+use Yiisoft\Router\RouteCollectorInterface;
+use Yiisoft\Yii\Debug\Collector\CollectorInterface;
+use Yiisoft\Yii\Debug\Collector\RouterCollector;
+
+final class RouterCollectorTest extends CollectorTestCase
+{
+    /**
+     * @var \PHPUnit\Framework\MockObject\MockObject|\Yiisoft\Router\RouteCollectorInterface
+     */
+    private $routeCollector;
+
+    private array $routes = [];
+
+    /**
+     * @param \Yiisoft\Yii\Debug\Collector\CollectorInterface|\Yiisoft\Yii\Debug\Collector\RouterCollector $collector
+     */
+    protected function collectTestData(CollectorInterface $collector): void
```

Comment:
Shouldn't we operate on `$collector` instead of `$this->routeCollector`?

Reply:
No, because `$collector` it's `Debug\RouteCollector` and `$this->routeCollector` is `Router\RouteCollectorInterface`